src/RoomInvite.js

```diff
@@ -46,9 +35,9 @@ export function inviteToRoom(roomId, addr) {
  * @param {string[]} addrs Array of strings of addresses to invite. May be matrix IDs or 3pids.
  * @returns {Promise} Promise
  */
-export function inviteMultipleToRoom(roomId, addrs) {
+function inviteMultipleToRoom(roomId, addrs) {
     const inviter = new MultiInviter(roomId);
-    return inviter.invite(addrs);
+    return inviter.invite(addrs).then(addrs => Promise.resolve({addrs, inviter}));
```

Comment:
`addrs` in the `then` closure is actually the completion states? Name it accordingly in this file?

Reply:
it is, yea. I thought I renamed it too :(